compiler/rustc_parse/src/parser/item.rs

```diff
@@ -81,6 +81,11 @@ impl<'a> Parser<'a> {
                         span,
                         "consider using `const` or `static` instead of `let` for global variables",
                     );
+                } else if self.parse_const_block(span, false).is_ok() {
+                    err.span_label(
+                        span,
```

Comment:
Also please use a structured suggestion (with `Applicability::MaybeIncorrect`)

```
err.span_suggestion_verbose(
    span.shrink_to_lo(),
    /* ... */
)
```

Reply:
The suggestion above this one for the `kw::Let` case should also be using something like that, right?

Reply:
Yea, though that one's harder because it has two choices. You can try to use [`span_suggestions`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/diagnostic/struct.Diag.html#method.span_suggestions) if you'd like.